Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -156,22 +146,22 @@ void FixedUpdate()
 		FollowBumper();
 
 		// Update
-		CurrentStateMethod();
+		_currentState.Update();
 
 		// Post-Update stuff
 		ConstrainBumper();
 		OrientSelf();
 	}
 
-	public void RegisterStateMethod( ActorStates state, ActorStateMethod method )
+	public void RegisterStateMethod( ActorStates state, PhysicsState method )
 	{
-		if ( _stateMethodMap.ContainsKey( state ) )
+		if ( _stateMap.ContainsKey( state ) )
 		{
-			_stateMethodMap[state] = method;
+			_stateMap[state] = method;
 		}
 		else
 		{
-			_stateMethodMap.Add( state, method );
+			_stateMap.Add( state, method );
 		}
 	}
```

Comment:
Are you saying add a comment to the `PhysicsState` class explaining its purpose? It's not clear from the line you commented on what you're referring to.